src/addon/addon_manager.cpp

```diff
@@ -445,18 +446,37 @@ AddonManager::enable_addon(const AddonId& addon_id)
         break;
     }
 
-    if (PHYSFS_mount(addon.get_install_filename().c_str(), mountpoint.c_str(), 1) == 0)
+    if (g_config->use_local_path)
     {
+      if (PHYSFS_mount(addon.get_install_filename().c_str(), mountpoint.c_str(), 0) == 0)
+      {
       log_warning << "Could not add " << addon.get_install_filename() << " to search path: "
                   << PHYSFS_getLastErrorCode() << std::endl;
+      }
+      else
+      {
+       if (addon.get_type() == Addon::LANGUAGEPACK)
+       {
+         PHYSFS_enumerate(addon.get_id().c_str(), add_to_dictionary_path, nullptr);
+       }
+      addon.set_enabled(true);
+      }
     }
-    else
-    {
-      if (addon.get_type() == Addon::LANGUAGEPACK)
+   else
+   {
+     if (PHYSFS_mount(addon.get_install_filename().c_str(), mountpoint.c_str(), 1) == 0)
       {
-        PHYSFS_enumerate(addon.get_id().c_str(), add_to_dictionary_path, nullptr);
+      log_warning << "Could not add " << addon.get_install_filename() << " to search path: "
+                  << PHYSFS_getLastErrorCode() << std::endl;
+      }
+      else
+      {
+       if (addon.get_type() == Addon::LANGUAGEPACK)
+        {
+         PHYSFS_enumerate(addon.get_id().c_str(), add_to_dictionary_path, nullptr);
+         }
+       addon.set_enabled(true);
       }
-      addon.set_enabled(true);
     }
```

Comment:
Indentation problem, brackets lining is correct but make sure it's +2 spaces after an open bracket and -2 before a closed one